---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

**[Download CV](http://cmanigbas.github.io/files/ManigbasCeline_CV_Nov2024.pdf)**

Education
======
* Massachusetts College of Liberal Arts
  * Bachelor of Science, Computer Science: Bioinformatics; summa cum laude

Work experience
======
* September 2022-current: Bioinformatician
  * Icahn School of Medicine at Mount Sinai, Department of Genetics and Genomics
  * Principle Investigator: Andrew Sharp
  * Developed custom pipelines for large-scale analyses to identify novel associations with >48,000 tandem repeats (TRs) and >30,000 clinically significant phenotypes in ~170k European samples from the UKBiobank.
  * Identified 47 TRs that showed causal associations with 73 traits based on fine-mapping and conditional analysis and were strongly enriched for functional repeats that impact local gene expression and DNA methylation using eQTLs and mQTLs. 

* Summer 2018 - Summer 2019: Research Intern
  * Cornell University, Boyce Thomoson Institute (BTI) Computational Biology Center
  * selected for [Plant Genome Research Program (NSF & USDA Funded REU)](https://btiscience.org/educational-programs/internships/interns/manigbas/)
  * Curated *A. syriaca* (common milkweed) genome assembly, including quality control, error correction, data cleaning, benchmarking, and functional annotation.
  * Performed gene family expansion analysis to identify likely ancestral genes and pathways crucial for the biosynthesis of cardiac glycosides and how they have evolved over time. 
    * [GenomicConsensus Arrow](https://github.com/PacificBiosciences/gcpp) (now deprecated), [Portcullis](https://github.com/EI-CoreBioinformatics/portcullis), [Mikado](https://mikado.readthedocs.io/en/stable/), [Scipio](http://www.webscipio.org), [OrthoFinder](https://github.com/davidemms/OrthoFinder), [InterProScan](https://github.com/ebi-pf-team/interproscan), [KinFin](https://github.com/DRL/kinfin)
  * Mentor: Susan Strickler

* Summer 2017: Research Intern
  * New York University, Department of Biochemistry and Molecular Pharmacology
  * Mentor: Timothy Cardozo
  * Ran molecular modelling simulations using crystal structures of the bacterophage T4 in ICM-Browser Pro.
  * Utilizing Monte Carlo sampling, modelled a previously unknown loop structure by which T4 highjacks the RNAP for *E. coli*    
  
Skills
======
* Proficiency in Programming languages:
  * python
  * R
  * Java
  * C++
  * Unix Shell (bash)
* Tools/analyses 
  * [REGENIE]()
  * [METAL]()
  * linear and logisitc regression
  * Genome-Wide association studies (GWAS) analysis
  * Expression quantitative trait loci (eQTL) analysis
  * Conditional Analysis
  * Fine-mapping [CAVIAR]()
* Project Management
* Team-oriented communications 

Relevant Coursework
======
* Biology
  * Genetics
  * Zoology
  * Cell and Molecular Biology, 
* Chemistry
  * Organic Chemsitry
  * Biochemistry
* Computer Science
  * Programming in Java
  * Programming in C++
  * Object Oriented Design
  * Database Developement
  * Data Structures and Algorithsm
  * Machine Learning
* Mathematics
  * Discrete Mathematics
  * Proof I & II
  * Biometry

Publications
======
  <ul>{% for post in site.publications reversed %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
Talks
======
  <ul>{% for post in site.talks reversed %}
    {% include archive-single-talk-cv.html  %}
  {% endfor %}</ul>
  
  
Service and leadership
======
* Assisted mentorship for intern in Icahn School of Medicine at Mount Sinai's Summer Undergraduate Research Program for Underserved Scholars (Summer 2023)  
* Assisted mentorship for intern in BTIâ€™s High School Research Program (Summer 2018) 
